src/lwc/geFormWidgetTokenizeCard/__mocks__/geFormWidgetTokenizeCard.js

```diff
@@ -0,0 +1,7 @@
+import { LightningElement, api } from 'lwc';
```

Comment:
Similar here, we shouldn't need a mock for geFormWidgetTokenizeCard in this spot. This mock could get used in other tests that depend on this component.